lib/ae_mdw/sync/async_tasks/derive_aex9_presence.ex

```diff
@@ -75,7 +75,10 @@ defmodule AeMdw.Sync.AsyncTasks.DeriveAex9Presence do
       [] ->
         contract_pk
         |> read_aex9_transfers_from_mb(kbi, mbi)
-        |> Enum.map(fn [_from_pk, to_pk, <<_amount::256>>] -> to_pk end)
+        |> Enum.flat_map(fn
+          [_from_pk, to_pk, <<_amount::256>>] -> [to_pk]
+          _other -> []
```

Comment:
Why would there be other aex9 transfers format? what are those other formarts?

Reply:
Same logic as here:
https://github.com/aeternity/ae_mdw/blob/78c664828a1445509d09375ad1b9427c3b345e57/lib/ae_mdw/db/sync/contract.ex#L64

There should be none because of this:
https://github.com/aeternity/ae_mdw/blob/78c664828a1445509d09375ad1b9427c3b345e57/lib/ae_mdw/sync/async_tasks/derive_aex9_presence.ex#L119

Bottom line this might need improvement but it is how transfers are filtered currently (as contract.ex existing code) outside the scope of the async task.
 
Without it, it gives: https://github.com/aeternity/ae_mdw/issues/364#issuecomment-981884473